src/models/event.ts

```diff
@@ -528,10 +534,7 @@ export class MatrixEvent extends EventEmitter {
             throw new Error("Attempt to decrypt event which isn't encrypted");
         }
 
-        if (
-            this.clearEvent && this.clearEvent.content &&
-            this.clearEvent.content.msgtype !== "m.bad.encrypted"
-        ) {
+        if (this.clearEvent?.content && this.clearEvent.content.msgtype !== "m.bad.encrypted") {
```

Comment:
Shouldn't `this.clearEvent?.content` now just be `this.clearEvent` given the presence of clearEvent is now the signal for whether something has or has not been decrypted - `this.clearEvent?.content` was the old signal for this